package.json

```diff
@@ -12,6 +12,7 @@
     "build": "npm run clean && npm run build-js && npm run build-sass && npm run favicons && npm run fonts && npm run images",
     "build-js": "mkdir -p dist/js/ && browserify --outfile dist/js/app.js js/*.js",
     "build-sass": "node-sass --include-path bower_components/ --output dist/css/ --source-map true --output-style compressed scss/",
+    "postbuild-sass": "postcss --use autoprefixer --replace dist/css/**/*.css",
```

Comment:
`node-sass` generates a single css file for us right?

does it make sense to use a pipe instead?

e.g. `node-sass [options without --output part] | postcss --use autoprefixer [some option for output location]`